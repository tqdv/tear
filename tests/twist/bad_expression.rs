use tear::twist;

fn bad_expression() {
	twist! { 1 => }
}

fn main () {}