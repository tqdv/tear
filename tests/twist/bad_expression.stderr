error: Expected either `$e` or `$e => $f` on the right-hand side, got: 1 =>
 --> $DIR/bad_expression.rs:4:2
  |
4 |     twist! { 1 => }
  |     ^^^^^^^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
